src/tmuxinator.ts

```diff
@@ -0,0 +1,141 @@
+const completionSpec: Fig.Spec = {
+  name: "tmuxinator",
+  description: "Tmuxinator is a ruby gem that allows you to easily manage tmux sessions by using yaml files to describe the layout of a tmux session, and open up that session with a single command",
+  subcommands: [
+    {
+      name: "commands",
+      description:
+        "Lists commands available in tmuxinator",
+    },
+    {
+      name: "completions",
+      description:
+        "Used for shell completion",
+    },
+    {
+      name: "copy",
```

Comment:
There are other subcommands missing aliases, could you add them?
```suggestion
      name: ["copy", "c", "cp"],
```